view/theme/frio/templates/wall_thread.tpl

```diff
@@ -59,9 +59,15 @@ as the value of $top_child_total (this is done at the end of this file)
 <span class="uriid" style="display: none;">{{$item.uriid}}</span>
 {{/if}}
 	<div class="media {{$item.shiny}}">
+	{{if !$item.parentguid==""}}
```

Comment:
Doesn't `{{if !$item.parentguid}}` work as well?

Reply:
It should be 
```suggestion
	{{if $item.parentguid}}
```

Reply:
seems it works